### PR TITLE
Fix prompt in example chemistry question

### PR DESCRIPTION
**Pull request by andino2019**
The changes proposed are to eliminate the confusion that may arise when a student is faced with having to provide an answer for "atomic number" when in reality, the question expects the student to provide an answer about the "number of atoms".

### CODE REVIEW

**Comment by andino2019** [comment]
I have reviewed and hereby sign the CLA

**Comment by andino2019** [comment]
The changes proposed are to eliminate the confusion that may arise when a student is faced with having to provide an answer for "atomic number" when in reality, the question expects the student to provide an answer about the "number of atoms".

File: exampleCourse/questions/workshop/PeriodicTable1/question.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <pl-question-panel>
-  <p> {{params.name}} has density $\rho = {{params.rho}} \rm g/cm^3$ and atomic weight $M = {{params.M}} \rm g/mol $</p>
-  <p> Determine the atomic number $N$.</p>
+  <p> {{params.name}} has density $\rho = {{params.rho}} \rm g/cm^3$ and a molar mass $M = {{params.M}} \rm g/mol $</p>
+  <p> Determine the number of atoms in 1 cm<sup>3</sup>.</p>
```
Comment (nwalters512) [inline exampleCourse/questions/workshop/PeriodicTable1/question.html:3]: The LaTeX formatting with `$ ... $` ensures consistent formatting with the previous paragraph. I added the text "of this element" to remove any ambiguity.

```suggestion
  <p> Determine the number of atoms in $1 \rm cm^3$ of this element.</p>
```

**Review by nwalters512 — APPROVED** [review]
Thanks again for bringing this to our attention!

**Pull request merged**